app/App.jsx

```diff
@@ -403,6 +405,19 @@ class App extends React.Component {
                                     path="/help/:path1/:path2/:path3"
                                     component={Help}
                                 />
+
+                                {/* CRYPTOBRIDGE: Support routes */}
+                                <Route
```

Comment:
Actually, for React Router 4 onwards, it is specified as e.g.:

    /support/ticketId?

instead of:

    /support(/:ticketId)

which is now deprecated. Thanks for the heads-up. Done it.